<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit:disabled{
	background-color: rgb(144, 144, 144)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="questioLabel">
     <property name="text">
      <string>Question:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="questionEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Confirm password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordConfirmEdit">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Submit</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
